src/app/about4dvd/about4dvd.component.html

```diff
@@ -0,0 +1,90 @@
+
+
+<div class="w3-container">
```

Comment:
This `about` screen totally messes up the viewport. It should bring up a window on top of the current screen, not move everything done. It's then confusing how your supposed to use it after that point as well. When I click outside of the about window, I'm still stuck in it.